overlay.yaml

```diff
@@ -39,15 +39,9 @@ actions:
   - target: $["paths"]["/appstore/apps/{app_id}/settings"]["patch"]
     update:
       "x-speakeasy-entity-operation": "AppStoreAppSettings#update"
-  - target: $["paths"]["/appstore/apps/{app_id}/settings"]["patch"]["parameters"][0]
-    update:
-      "x-speakeasy-match": "id"
   - target: $["paths"]["/appstore/apps/{app_id}/settings"]["get"]
     update:
       "x-speakeasy-entity-operation": "AppStoreAppSettings#read"
-  - target: $["paths"]["/appstore/apps/{app_id}/settings"]["get"]["parameters"][0]
-    update:
-      "x-speakeasy-match": "id"
```

Comment:
@niamhdurfee let me know if you agree with this. I don't think we should be matching `id` -> `app_id`, because my understanding is we _want_ the caller to specify an `app_id` when writing a `AppStoreAppSettings` resource.

Correct me if I'm wrong though, I thought we were moving all of this configuration under the `App` resource, which would make `AppStoreAppSettings` moot